charts/opentelemetry-operator/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-operator
-version: 0.71.2
+version: 0.72.0
```

Comment:
you can change this to only bump the patch version

Reply:
done

Reply:
Not sure how to fix the _Test Operator Charts_ jobs.
Any suggestion?